python/lsst/ts/MTAirCompressor/aircompressor_model.py

```diff
@@ -114,6 +114,11 @@ async def set_register(self, address, value, error_status):
             address, [value], slave=self.unit
         )
         if isinstance(result, pymodbus.pdu.ExceptionResponse):
+            if result.function_code == 144:
```

Comment:
Maybe a more meaningful name instead of magic number of 144. Or put it into the `constant.py` or `enums.py` or something similar.

Reply:
Changed to exception code 4, added comment. Also started issue for pymodbus, as those codes are Modbus-wide:
https://github.com/pymodbus-dev/pymodbus/issues/1300